packages/ove-service-layout/docs/WORKING_EXAMPLE.md

```diff
@@ -0,0 +1,105 @@
+# Working example
+
+```json
+{
+  "hasVideos": false,
```

Comment:
please get rid of hasVideos. It is some silly parameter carried forward from the past. I am to be blamed for this.

Reply:
The parameter is ignored by the layout service, but if this not standard practice anymore, I can remove it.